sauce/features/accounts/toggle-transaction-filters/index.js

```diff
@@ -0,0 +1,100 @@
+import { Feature } from 'toolkit/core/feature';
+import * as toolkitHelper from 'toolkit/helpers/toolkit';
+
+export class ToggleTransactionFilters extends Feature {
+  injectCSS() {
+    return require('./index.css');
+  }
+
+  shouldInvoke() {
+    return toolkitHelper.getCurrentRouteName().indexOf('account') > -1;
+  }
+
+  invoke() {
```

Comment:
probably going to want to add an `onRouteChanged` listener for this...

right now, if you refresh YNAB on the budget page I don't think this will ever work.

We're adding the listeners on the body of the page only if we're invoked (first loaded) on the account page.

Another, probably better option for this is to add the click listeners directly to the elements we're putting into the dom and then inside `initToggleButtons` just check if the buttons already exist.